core/src/Exceptions/CoreErrorMessages.cs

```diff
@@ -51,7 +51,8 @@ internal class CoreErrorMessages
         public const string UnsupportedAuthorityValidation =
             "Authority validation is not supported for this type of authority";
 
-        public const string AuthenticationCanceled = "User canceled authentication";
+        public const string AuthenticationCanceled = "User canceled authentication. On an Android device, this could be due to the system browser." +
+            " See https://aka.ms/msal-net-system-browsers for more information.";
```

Comment:
On a few of the browsers that do not support custom tabs, after entering creds, we do not get redirected back to the app, and the user needs to leave the app and then go back to it. when that happens, a User Canceled Auth exception is returned, so I thought to add a bit to the error message to handle that case. Not sure where else to put it, as the app is basically frozen, but wanted to keep the message short, as it's android specific. thoughts?

Reply:
Good enough as long as the aka.ms link is there and it shows details on capabilities the browser needs to have and some examples of browsers you have validated works and browsers you have seen fail. (I suggest simply re-using the table you have here)

---
In reply to: [227610125](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227610125) [](ancestors = 227610125)

Reply:
I was going to add the table and maybe something else (a cat pic or two). 

---
In reply to: [227648339](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1347#discussion_r227648339) [](ancestors = 227648339,227610125)